package/MDAnalysis/coordinates/base.py

```diff
@@ -1025,6 +1089,32 @@ def add_auxiliary(self, auxname, auxdata, format=None, **kwargs):
         self._auxs[auxname] = aux
         self.ts = aux.update_ts(self.ts)
 
+    def _add_aux_edr(self,
```

Comment:
Again if we provide a consistent interface between the two current readers then you should just be able to have one _add_aux method. There are probably complications I am missing, but seems a cleaner way than providing a different attatchment point for each format. The `.attatch ` idea suggested above would fix this but open to discussion.